src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -186,42 +186,113 @@ private void userOnDesktopMustBeLoggedInToUsePostCreator() {
   /**
    * This test covers all situations when post cannot be added (submit button is disabled).
    * <p>
-   * | Category | Title | Description |
-   * |          |     x |             |
-   * |          |     x |           x |
-   * |          |       |           x |
-   * |        x |       |             |
-   * |        x |     x |             |
+   * | Category | Title | Description | Open Graph | Content Image |
+   * |          |       |             |            |               |
+   * |          |     x |             |            |               |
+   * |          |       |           x |            |               |
+   * |          |       |             |          x |               |
+   * |          |       |             |            |             x |
+   * |          |     x |           x |            |               |
+   * |          |     x |             |          x |               |
+   * |          |     x |             |            |             x |
+   * |        x |       |             |            |               |
+   * |        x |     x |             |            |               |
+   * |        x |       |           x |            |               |
+   * |        x |       |             |          x |               |
+   * |        x |       |             |            |             x |
+   *
+   *
    * <p>
    * * x - means category was selected or text was added
    */
-  private void assertThatPostWithoutSelectedCategoryAndDescriptionCannotBeAdded(PostsCreator postsCreator) {
-    postsCreator.click()
-        .closeGuidelinesMessage();
+  private void assertThatPostWithoutCategoryOrAnyContentCannotBeAdded(BasePostsCreator postsCreator) {
+    postsCreator.click().closeGuidelinesMessage();
     Assertion.assertFalse(postsCreator.isPostButtonActive());
 
     postsCreator.addTitleWith(TextGenerator.defaultText());
-    Assertion.assertFalse(postsCreator.isPostButtonActive(),
-        "User should not be able to add post with only title filled.");
+    Assertion.assertFalse(
+        postsCreator.isPostButtonActive(),
+        "User should not be able to add post with only title."
+    );
+    postsCreator.clearTitle();
 
     postsCreator.addDescriptionWith(TextGenerator.defaultText());
-    Assertion.assertFalse(postsCreator.isPostButtonActive(),
-        "User should not be able to add post with title and description filled.");
+    Assertion.assertFalse(
+        postsCreator.isPostButtonActive(),
+        "User should not be able to add post with only description."
+    );
+    postsCreator.clearDescription();
 
-    postsCreator.clearTitle();
-    Assertion.assertFalse(postsCreator.isPostButtonActive(),
-        "User should not be able to add post with only description filled.");
+    postsCreator.addDescriptionWithLink("http://fandom.wikia.com");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "http://fandom.wikia.com" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)